client/apps/dashboard/src/app/pages/customers/pages/single-page/customers-single-page.component..html

```diff
@@ -31,8 +31,16 @@
     </mat-form-field>
   </div>
   <div class="col-12 ta-right">
+    <ng-container class="m-b-s">
+      <button
+        mat-flat-button
+        (click)="cancel()">
+        Cancel
+      </button>
+    </ng-container>
+
     <button
-      (click)="basicSubmit()"
+      (click)="save(basicInfoForm.getRawValue())"
```

Comment:
The form is called `form` not `basicInfoForm`

```suggestion
      (click)="save(form.getRawValue())"
```